cypress/integration/carry_okay_spec.js

```diff
@@ -105,20 +106,27 @@ describe('CarryOkay', () => {
   describe('Song Book', () => {
 
   beforeEach(() => {
-    cy.intercept('/genres', {fixture:"genre_data.json"})
-    cy.intercept('/songs', {fixture:"song_data.json"})
+    cy.intercept( 'https://carryokay-server.herokuapp.com/genres', {fixture:"genre_data.json"})
+    cy.intercept('https://carryokay-server.herokuapp.com/songs', {fixture:"song_data.json"})
     cy.visit('http://localhost:3000');
     cy.get('[data-cy=song-book-nav]').click();
     });
 
   it('should display Song Book page components', () => {
-    cy.get('[data-cy=song-book-title]').contains('Song Book');
+    cy.get('[data-cy=home-nav]').should('exist');
     cy.get('[data-cy=my-songs-nav]').contains("My Songs");
     cy.get('[data-cy=song-book-nav]').contains("Song Book");
     cy.get('[data-cy=search-form]').should('exist');
     cy.get('[data-cy=song-card]').should("have.length", 2);
     cy.get('[data-cy=song-card]').eq(0).contains("Radiohead");
     cy.get('[data-cy=song-card]').eq(1).contains("Usher");
+    cy.get('[data-cy=song-card]').first().should("contain", "Jigsaw falling into place")
+    .and("contain", "Radiohead")
+    .and("contain","Electronica")
+    .and("contain","Rock")
+  cy.get('[data-cy=album-img]').first().should('exist');
+  cy.get('[data-cy=song-card-btn]').first().should('exist');
+  cy.get('[data-cy=microphone-icon]').should('exist');
   });
```

Comment:
I wonder if we should create a map and loop through each song

Reply:
ok yeah, I can implement that